Clicker/ViewControllers/CardController.swift

```diff
@@ -202,51 +238,25 @@ class CardController: UIViewController {
         }
         // TODO: Add logic for updating countLabel to display current question # / total num questions
```

Comment:
Is this still an active TODO? If so lets make it an issue and implement that before launch.